<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>225</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-image: url(:/res/img/fon2.jpg);}

QPushButton {
color: rgb(255, 255, 255);
     border-radius: 30px;
     background-color: rgb(81, 78, 120);
     min-width: 80px;
 } 
 QPushButton:pressed {
	background-color: rgb(104, 93, 145);
 }

 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }
QPushButton::hover {
    background-color: rgb(115, 104, 161);
}

QLabel{color: rgb(50, 48, 74);}



QLineEdit{
border: 1px solid /*rgb(89, 145, 166);*/ rgb(197, 197, 197);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}
QLineEdit:hover{
border: 1px solid rgb(115, 104, 161);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}
QLineEdit:focus{
border: 2px solid rgb(106, 209, 208);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="weightG">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="onward_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
